Lib9c/TableData/StakeRegularRewardSheet.cs

```diff
@@ -32,35 +38,45 @@ public override int GetHashCode()
             {
                 unchecked
                 {
-                    return (ItemId * 397) ^ Rate;
+                    return (ItemId * 397) ^ RateOrCount ^ Type.GetHashCode();
                 }
             }
 
             public readonly int ItemId;
-            public readonly int Rate;
+            public readonly int RateOrCount;
+            public readonly RewardType Type;
 
             public RewardInfo(params string[] fields)
             {
                 ItemId = ParseInt(fields[0]);
-                Rate = ParseInt(fields[1]);
+                RateOrCount = ParseInt(fields[1]);
+                Type = fields.Length >= 3 && Enum.TryParse(fields[2], true, out RewardType type)
+                    ? type
+                    : RewardType.Arithmetic;
             }
 
-            public RewardInfo(int itemId, int rate)
+            public RewardInfo(int itemId, int rateOrCount, RewardType type)
             {
                 ItemId = itemId;
-                Rate = rate;
+                RateOrCount = rateOrCount;
+                Type = type;
             }
 
             public RewardInfo(Dictionary dictionary)
```

Comment:
There is no case RewardInfo serialize/deserialize to `IValue`.

Reply:
I removed it in [dfc9470](https://github.com/planetarium/lib9c/pull/1102/commits/dfc9470600da5d781191357f8599aaa05b43da7a)